view/scn-social-auth/user/login.phtml

```diff
@@ -1,10 +1,6 @@
 <?php
-$socialSignIn = true;
+echo '<h1>' . $this->translate('Social Sign In') . '</h1>';
```

Comment:
Fair, but it's a behavior change I'm not willing to introduce in the 1.x.x version.